tests/unit/popover.spec.ts

```diff
@@ -315,24 +314,40 @@ describe('Popover', function() {
     it('should emit "closed" on click away', async function() {
       const anotherelement = wrapper.find('#anotherelement');
       await anotherelement.trigger('click');
-      await wrapper.vm.$nextTick();
+
       expect(popoverWrapper.emitted().closed.length).toEqual(1);
     });
 
     it('should not emit "closed" on click on it', async function() {
       await popoverWrapper.trigger('click');
-      await wrapper.vm.$nextTick();
+
       expect(popoverWrapper.emitted().closed).toBeUndefined();
     });
   });
 
   describe('destroyPositioning', function() {
     it('should call destroyPosition when visible becomes false', async function() {
       createWrapper({ props: { visible: true } });
-      await popoverWrapper.vm.$nextTick();
+
       const destroyPositioningSpy: any = jest.spyOn(popoverWrapper.vm as any, 'destroyPositioning');
       popoverWrapper.setProps({ visible: false });
       expect(destroyPositioningSpy).toHaveBeenCalled();
     });
+
+    it('should call destroyPosition at destroy if visible', async function() {
+      createWrapper({ props: { visible: true } });
+
+      const destroyPositioningSpy: any = jest.spyOn(popoverWrapper.vm as any, 'destroyPositioning');
+      popoverWrapper.destroy();
```

Comment:
Outside this PR scope: should we not destroy wrappers for other tests too to avoid memory leaks?
Maybe it's not necessary due to the way Jest runs tests, compared to mocha.
@JeremyPinhel I'd like your opinion on this, thanks! :)